src/Worker.Extensions.DurableTask/Execution/DurableTaskRegistryExtensions.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Microsoft.DurableTask;
+using Microsoft.DurableTask.Entities;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.DurableTask.Execution;
+
+/// <summary>
+/// Extension methods for <see cref="DurableTaskRegistry"/>.
+/// </summary>
+internal static class DurableTaskRegistryExtensions
+{
+    private static readonly BindingFlags Flags = BindingFlags.Instance | BindingFlags.NonPublic;
+
+    /// <summary>
+    /// Gets the registered orchestrators from the registry.
+    /// </summary>
+    /// <param name="registry">The registry to get orchestrators from.</param>
+    /// <returns>The current registered orchestrators.</returns>
+    public static IEnumerable<KeyValuePair<TaskName, Func<IServiceProvider, ITaskOrchestrator>>> GetOrchestrators(
+            this DurableTaskRegistry registry)
+    {
+        if (registry is null)
+        {
+            throw new ArgumentNullException(nameof(registry));
+        }
+
+        // TODO: expose this in durabletask-dotnet
```

Comment:
When do we plan on taking care of this TODO? Is there a GitHub issue that tracks it?

Reply:
No issues. They will need to be opened.